be/src/exec/CMakeLists.txt

```diff
@@ -183,6 +183,9 @@ set(EXEC_FILES
     pipeline/set/intersect_output_source_operator.cpp
 )
 
+# simdjson Runtime CPU Detection: https://github.com/simdjson/simdjson/blob/master/doc/implementation-selection.md#runtime-cpu-detection
+set_source_files_properties(vectorized/json_scanner.cpp PROPERTIES COMPILE_FLAGS -mno-avx)
```

Comment:
I have no idea for this flag with this comment.
I have some question about this Detection.
When does the detection work? At compile time or at runtime?
If it works at compile time, StarRocks is compiled in avx512 machine, will it work correctly in the avx2 machine?

Reply:
1. The detection works at compile and runtime both. The detection at compile determines implementation available for runtime. In general, the simdjson lib would choose the best implementation at runtime automatically.   
2.  The machine with avx512 could also be with avx512. Besides, the simdjson do not rely on avx512.
3. To be clearer, the comment is updated.

Reply:
I see, thanks for your reply.